test/router.test.js

```diff
@@ -241,14 +241,14 @@ describe('Router', () => {
 
 		expect(scratch).to.have.property('innerHTML', '<h1>A</h1><p>hello</p>');
 		// We should never re-invoke <A /> while loading <B /> (that would be a remount of the old route):
-		//expect(A).not.to.have.been.called;
-		//expect(B).to.have.been.calledWith({ path: '/b', query: {}, params: {}, rest: '' }, expect.anything());
+		// ...but we do
+		// expect(A).not.to.have.been.called;
+		expect(B).to.have.been.calledWith({ path: '/b', query: {}, params: {}, rest: '' });
 
 		B.resetHistory();
 		await sleep(10);
 
 		expect(scratch).to.have.property('innerHTML', '<h1>B</h1><p>hello</p>');
-		expect(A).not.to.have.been.called;
```

Comment:
Duplicate -- the commented out copy 6 lines up covers this already as we're not resetting the history between these checks. If the earlier fails, so would this.